ansible_collections/arista/avd/plugins/action/batch_template.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
without annotations it will fail in 3.8 I think

Reply:
it would not, because we don't use pipe anywhere